src/js/jsx/sections/transform/TransformPanel.jsx

```diff
@@ -66,15 +98,67 @@ define(function (require, exports, module) {
+                                    L18.000,9.000 L18.000,7.000 L16.000,7.000 L16.000,5.000 ZM20.000,6.000
+                                    L20.000,3.000 L17.000,3.000 L17.000,6.000 L20.000,6.000 ZM20.000,13.000
+                                    L20.000,10.000 L17.000,10.000 L17.000,13.000 L20.000,13.000 ZM17.000,17.000
+                                    L17.000,20.000 L20.000,20.000 L20.000,17.000 L17.000,17.000 ZM9.000,9.000
+                                    L14.000,9.000 L14.000,14.000 L9.000,14.000 L9.000,9.000 ZM10.000,13.000
+                                    L13.000,13.000 L13.000,10.000 L10.000,10.000 L10.000,13.000 Z" />
+                                    <rect className={getReferenceClasses("rb")} id="rb"
+                                        x="16" y="16" width="5" height="5"
+                                        onClick={this._handleReferenceClick.bind(this, "rb")}/>
+                                    <rect className={getReferenceClasses("mb")} id="mb"
+                                        x="9" y="16" width="5" height="5"
+                                        onClick={this._handleReferenceClick.bind(this, "mb")}/>
+                                    <rect className={getReferenceClasses("lb")} id="lb"
+                                        x="2" y="16" width="5" height="5"
+                                        onClick={this._handleReferenceClick.bind(this, "lb")}/>
+                                    <rect className={getReferenceClasses("rc")} id="rc"
+                                        x="16" y="9" width="5" height="5"
+                                        onClick={this._handleReferenceClick.bind(this, "rc")}/>
+                                    <rect className={getReferenceClasses("mc")} id="mc"
+                                        x="9" y="9" width="5" height="5"
+                                        onClick={this._handleReferenceClick.bind(this, "mc")}/>
+                                    <rect className={getReferenceClasses("lc")} id="lc"
+                                        x="2" y="9" width="5" height="5"
+                                        onClick={this._handleReferenceClick.bind(this, "lc")}/>
+                                    <rect className={getReferenceClasses("rt")} id="rt"
+                                        x="16" y="2" width="5" height="5"
+                                        onClick={this._handleReferenceClick.bind(this, "rt")}/>
+                                    <rect className={getReferenceClasses("mt")} id="mt"
+                                        x="9" y="2" width="5" height="5"
+                                        onClick={this._handleReferenceClick.bind(this, "mt")}/>
+                                    <rect className={getReferenceClasses("lt")} id="lt"
+                                        x="2" y="2" width="5" height="5"
+                                        onClick={this._handleReferenceClick.bind(this, "lt")}/>
```

Comment:
Not sure if I like the implementation by pasting the entire SVG, but we should encapsulate this as ReferencePoint component, just like Position, Size components.